EqualityInformationApi.Tests/AwsMockWebApplicationFactory.cs

```diff
@@ -0,0 +1,71 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.DynamoDBv2.Model;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Hackney.Core.DynamoDb;
+
+namespace EqualityInformationApi.Tests
+{
+    public class AwsMockWebApplicationFactory<TStartup>
```

Comment:
Can we rename the class to `DynamodbMockWebApplicationFactory` if we are not going to use SNS